sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/Models/SignalRCreateOrUpdateProperties.cs

```diff
@@ -68,15 +63,6 @@ public SignalRCreateOrUpdateProperties(string hostNamePrefix = default(string),
         /// </summary>
         partial void CustomInit();
 
-        /// <summary>
-        /// Gets or sets prefix for the hostName of the SignalR service.
-        /// Retained for future use.
-        /// The hostname will be of format:
-        /// &amp;amp;lt;hostNamePrefix&amp;amp;gt;.service.signalr.net.
-        /// </summary>
-        [JsonProperty(PropertyName = "hostNamePrefix")]
-        public string HostNamePrefix { get; set; }
```

Comment:
This is a breaking change - you should not make breaking changes from the previous, stable api.

Reply:
This change is covered by the swagger PR above.